docs/feature_ps2_mouse.md

```diff
@@ -155,6 +155,27 @@ In your keyboard config.h:
 #endif
 ```
 
+### RP2040 PIO Version :id=rp2040-pio-version
+
+This uses the RP2040's hardware feature called PIO and is therefore only
+available on boards that are based on the RP2040 MCU.
+
+The pin for the data signal must be followed directly by the pin for the clock
+within the GPIO pin numbering scheme. This is required due to the design of the
+PIO state machine code.
```

Comment:
````suggestion
The `PIO` subsystem is a Raspberry Pi RP2040 specific implementation, using the integrated PIO peripheral and is therefore only available on this MCU.

There are strict requirements for pin ordering but any pair of GPIO pins can be used. The GPIO used for clock must be directly after data, see the included info.json snippet for an example of correct order.

You may optional switch the PIO peripheral used with the following define:
```c
#define PS2_PIO_USE_PIO1 // Force the usage of PIO1 peripheral, by default the PS2 implementation uses the PIO0 peripheral
```
````

Reply:
Okay this is much nicer, thank you for taking time to improve it. I'm only not sure about the "PIO subsystem" term, since what we describe is just one state machine implementation that is using the PIO subsystem of an RP2040. Maybe just "This PS/2 driver is a..."?

Reply:
Just trying to keep language consistent accross the docs, it's a copy and paste from another section. https://docs.qmk.fm/#/serial_driver?id=the-pio-driver

Reply:
Ah I see! Consistency is good.